generate_chip.py

```diff
@@ -889,71 +1010,126 @@ def _uuid(identifier: str) -> str:
+                BodyDimensions(3.5, 2.8, 2.1, 0.8, 2.2),
+                footprints={
+                    'A': FootprintDimensions(2.21, 2.35, 0.92),
+                    'B': FootprintDimensions(1.80, 2.23, 1.12),
+                    'C': FootprintDimensions(1.42, 2.13, 1.28),
+                },
+                meta={'eia': '3528-21', 'kemet': 'B', 'avx': 'B'},
+            ),
+            ChipConfig(
+                '',
+                BodyDimensions(6.0, 3.2, 1.5, 1.3, 2.2),
+                footprints={
+                    'A': FootprintDimensions(2.77, 2.35, 2.37),
+                    'B': FootprintDimensions(2.37, 2.23, 2.57),
+                    'C': FootprintDimensions(1.99, 2.13, 2.73),
+                },
+                meta={'eia': '6032-15', 'kemet': 'U', 'avx': 'W'},
+            ),
+            ChipConfig(
+                '',
+                BodyDimensions(6.0, 3.2, 2.8, 1.3, 2.2),
+                footprints={
+                    'A': FootprintDimensions(2.77, 2.35, 2.37),
+                    'B': FootprintDimensions(2.37, 2.23, 2.57),
+                    'C': FootprintDimensions(1.99, 2.13, 2.73),
+                },
+                meta={'eia': '6032-28', 'kemet': 'C', 'avx': 'C'},
+            ),
+            ChipConfig(
+                '',
+                BodyDimensions(7.3, 6.0, 3.8, 1.3, 4.1),
+                footprints={
+                    'A': FootprintDimensions(2.77, 4.25, 3.68),
+                    'B': FootprintDimensions(2.37, 4.13, 3.87),
+                    'C': FootprintDimensions(1.99, 4.03, 4.03),
+                },
+                meta={'eia': '7360-38', 'kemet': 'E', 'avx': 'V'},
+            ),
+            ChipConfig(
+                '',
+                BodyDimensions(7.3, 4.3, 2.0, 1.3, 2.4),
+                footprints={
+                    'A': FootprintDimensions(2.77, 2.55, 3.67),
+                    'B': FootprintDimensions(2.37, 2.43, 3.87),
+                    'C': FootprintDimensions(1.99, 2.33, 4.03),
+                },
+                meta={'eia': '7343-20', 'kemet': 'V', 'avx': 'Y'},
+            ),
+            ChipConfig(
+                '',
+                BodyDimensions(7.3, 4.3, 3.1, 1.3, 2.4),
+                footprints={
+                    'A': FootprintDimensions(2.77, 2.55, 3.67),
+                    'B': FootprintDimensions(2.37, 2.43, 3.87),
+                    'C': FootprintDimensions(1.99, 2.33, 4.03),
+                },
+                meta={'eia': '7343-31', 'kemet': 'D', 'avx': 'D'},
+            ),
+            ChipConfig(
+                '',
+                BodyDimensions(7.3, 4.3, 4.3, 1.3, 2.4),
+                footprints={
+                    'A': FootprintDimensions(2.77, 2.55, 3.67),
+                    'B': FootprintDimensions(2.37, 2.43, 3.87),
+                    'C': FootprintDimensions(1.99, 2.33, 4.03),
+                },
+                meta={'eia': '7343-43', 'kemet': 'X', 'avx': 'E'},
+            ),
```

Comment:
I think for this code block I'd also disable the formatter, though its not that bad here (just much longer than before).